engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -397,6 +398,9 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
       } else {
         goto(WorkflowExecutionAbortedState)
       }
+    case Event(EngineStatsActor.JobCountQuery, data) =>
+      sender ! EngineStatsActor.JobCount(data.backendJobExecutionActors.size)
```

Comment:
Right now it seems like job count is the # of backendJobExecutionActors, but it's possible that if a user has caching on, they might want to know the # of jobs that are busy caching, otherwise their job count will appear 0 and give this false sense of no work being done. Is it possible to return the count of backend job execution as well as caching actors? though not sure if we have a list of caching actors anywhere right now? If this is outside the scope of the original ticket, we can file it for a future change.